pdp_util/raster.py

```diff
@@ -86,12 +86,21 @@ def __call__(self, environ, start_response):
                 charset="utf-8",
             )
             return res(environ, start_response)
-
+        elif not req.path_info.split(".")[-1].endswith("nc"):
```

Comment:
I appreciate your documentation on the fancy new functions you added. Might be nice to also add a comment saying what the three branches of this if do, in high level terms. My guess is file list, .dds or .das metadata, and actual data?

Reply:
Just added comments.